examples/pdm/example_lock.bzl

```diff
@@ -1062,6 +1094,85 @@ def targets():
         wheel = "@example_lock_wheel_six_1.16.0_py2.py3_none_any//file",
     )
 
+    pycross_wheel_library(
+        name = "snowballstemmer_2.2.0",
+        wheel = "@example_lock_wheel_snowballstemmer_2.2.0_py2.py3_none_any//file",
+    )
+
+    _sphinx_7_2_6_deps = [
+        ":alabaster_0.7.13",
+        ":babel_2.13.1",
+        ":docutils_0.20.1",
+        ":imagesize_1.4.1",
+        ":jinja2_3.1.2",
+        ":packaging_23.2",
+        ":pygments_2.16.1",
+        ":requests_2.31.0",
+        ":snowballstemmer_2.2.0",
+        ":sphinxcontrib_applehelp_1.0.7",
+        ":sphinxcontrib_devhelp_1.0.5",
+        ":sphinxcontrib_htmlhelp_2.0.4",
+        ":sphinxcontrib_jsmath_1.0.1",
+        ":sphinxcontrib_qthelp_1.0.6",
+        ":sphinxcontrib_serializinghtml_1.1.9",
+    ]
+
+    pycross_wheel_library(
+        name = "sphinx_7.2.6",
+        deps = _sphinx_7_2_6_deps,
+        wheel = "@example_lock_wheel_sphinx_7.2.6_py3_none_any//file",
+    )
+
+    _sphinxcontrib_applehelp_1_0_7_deps = [
+    ]
```

Comment:
Can we filter these empty lists out? I think in the normal case, if a package has no dependencies we don't have an empty list.

Reply:
yep. just wanted to go with the quickest implementation to get your feedback first :)

Reply:
done